.gitignore

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Remove this empty line no `1`

```suggestion
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I have made changes to the best of knowledge. Please check and assist. thank you.